src/blender_mcp/server.py

```
@@ -286,42 +286,39 @@ def get_object_info(ctx: Context, object_name: str) -> str:
-        temp_dir = tempfile.gettempdir()
-        temp_path = os.path.join(temp_dir, f"blender_screenshot_{os.getpid()}.png")
         
         result = blender.send_command("get_viewport_screenshot", {
             "max_size": max_size,
-            "filepath": temp_path,
+            "reserve": reserve,
             "format": "png"
         })
         
         if "error" in result:
             raise Exception(result["error"])
```

Ruff (0.15.1)

[warning] Abstract `raise` to an inner function

(TRY301)